extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -7,9 +7,55 @@ define('ext.wikia.adEngine.template.porvata', [
 	'ext.wikia.adEngine.video.player.ui.videoInterface',
 	'ext.wikia.adEngine.video.videoSettings',
 	'wikia.document',
+	'wikia.log',
+	'wikia.window',
 	require.optional('ext.wikia.adEngine.mobile.mercuryListener')
-], function (DOMElementTweaker, porvata, googleIma, uiTemplate, videoInterface, videoSettings, doc, mercuryListener) {
+], function (
+	DOMElementTweaker,
+	porvata,
+	googleIma,
+	uiTemplate,
+	videoInterface,
+	videoSettings,
+	doc,
+	win,
+	log,
+	mercuryListener
+) {
 	'use strict';
+	var logGroup = 'ext.wikia.adEngine.template.porvata';
+
+	function hideOtherBidsForVeles(params) {
```

Comment:
We should consider creating `veles` template. It seems not needed in general `porvata` template